packages/paste-core/components/combobox/stories/index.stories.tsx

```diff
@@ -484,4 +487,46 @@ storiesOf('Components|Combobox', module)
         itemToString={(item: GroupedItem) => (item ? item.label : null)}
       />
     );
+  })
+  .add('Combobox - Listbox zIndex', () => {
+    const [selectValue, setSelectValue] = React.useState('');
+    const selectID = useUIDSeed();
```

Comment:
nit: we should call this `seed` or `idSeed` for clarity, or you can use `useUID` which would return the `selectID` directly

Reply:
Good catch. No reason to use `useUIDSeed` here since I'm just generating one ID.